src/main/java/emu/grasscutter/server/packet/recv/HandlerEvtDoSkillSuccNotify.java

```diff
@@ -15,6 +18,7 @@ public void handle(GameSession session, byte[] header, byte[] payload) throws Ex
         int skillId = notify.getSkillId();
         int casterId = notify.getCasterId();
 
+        HSDAbilityManager.skillId = skillId;
```

Comment:
a static property here would not work in the long run, pass the ID through to the managers.